public/docs/js/latest/guide/forms.jade

```diff
@@ -348,44 +373,33 @@ figure.image-display
 
 .l-main-section
 :marked
-  ## Track change-state and validity with **ngControl**
+  ## Track change-state and validity with **ngModel**
 
   A form isn't just about data binding. We'd also like to know the state of the controls on our form.
-  The `NgControl` directive keeps track of control state for us.
 
-.callout.is-helpful
-  header NgControl requires Form
-  :marked
-    The `NgControl` is one of a family of `NgForm` directives that can only be applied to
-    a control within a `<form`> tag.
-:marked
-  Our application can ask an `NgControl` if the user touched the control,
-  if the value changed, or if the value became invalid.
+  Using `ngModel` in a form gives us more than just two way data binding. It also tells us if the user touched the control, if the value changed, or if the value became invalid.
```

Comment:
This line 380 is ending with "if the".  It is going to be followed with "value changed, or if the value became invalid" as seen in line 363 above?